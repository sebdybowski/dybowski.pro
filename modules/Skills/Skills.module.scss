@use 'styles/variables.scss' as *;

.Skills {
    text-align: center;
    border-radius: 5px;

    & article {
        &:nth-child(2) {
            border: 2px solid;
            border-image-source: $brand-bg;
            border-image-slice: 1;
            position: relative;
            z-index: 100;
        
            @media screen and (min-width: $breakpoint-md) {
                transform: scale(1.1);
            }
        }

        & ul {
            text-align: left;

            & strong {
                float: right;
            }
        }
    }
}
