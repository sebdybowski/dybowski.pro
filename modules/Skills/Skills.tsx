import { faAngular, faAws, faCss3Alt, faFigma, faHtml5, faJava, faJs, faNode, faReact, faVuejs } from '@fortawesome/free-brands-svg-icons';
import { faCompassDrafting, faObjectGroup } from '@fortawesome/free-solid-svg-icons';
import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
import React, { FC } from 'react';

import { Column, Container, Heading } from '../../components/';

import styles from './Skills.module.scss';

interface SkillsProps { }

const Skills: FC<SkillsProps> = () => (
  <div className={styles.Hero} data-testid="Skills">
    <div className='container'>
      <br />
      <div className='grid'>
        <h2>Skills</h2>
      </div>
      <div className='grid'>
        <article>
          <header>Back-end</header>
          <ul>
            <li>Java</li>
          </ul>
        </article>
        <article>
          <header>Front-end</header>
          <ul>
            <li>Javascript <progress value="90" max="100"></progress></li>
            <li>
              <FontAwesomeIcon
                size='1x'
                icon={faReact}
              />
              React <strong>90%</strong>
              <progress value="100" max="100"></progress></li>
          </ul>
        </article>
        <article>
          <header>UI/UX</header>
          <ul>
            <li>React</li>
          </ul>
        </article>
      </div>
    </div>
  </div>
);

export default Skills;
