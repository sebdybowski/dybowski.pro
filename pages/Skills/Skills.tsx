import React, { FC } from 'react';
import styles from './Skills.module.css';

interface SkillsProps {}

const Skills: FC<SkillsProps> = () => (
  <div className={styles.Skills} data-testid="Skills">
    Skills Component
  </div>
);

export default Skills;
