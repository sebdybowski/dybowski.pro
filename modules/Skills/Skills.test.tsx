import React from 'react';
import { render } from '@testing-library/react';

import Skills from './Skills';

describe('<Skills />', () => {
  test('it should mount', () => {
    const { asFragment } = render(<Skills />);
    
    expect(asFragment()).toMatchSnapshot();
  });
});
