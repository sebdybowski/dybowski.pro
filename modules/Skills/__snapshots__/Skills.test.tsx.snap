// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Skills /> it should mount 1`] = `
<DocumentFragment>
  <div
    class="Hero"
    data-testid="Skills"
  >
    Skills Component
  </div>
</DocumentFragment>
`;
